joplin/templates/wagtailadmin/shared/field.html

```diff
@@ -0,0 +1,21 @@
+<!--
+Joplin edited from Original file: https://github.com/wagtail/wagtail/blob/master/wagtail/admin/templates/wagtailadmin/shared/field.html
+-->
```

Comment:
We just need this template so that we can remove the build in error messages.

Reply:
thanks for including a link to the original version